src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
I really like that layout, especially the pixelated font :laughing: 

But in all seriousness I get where you are coming from with the whole "its 40 pixels" thing.  How about a compromise, there was another PR a while back where we added a setting option but did not add a UI for it.

What if we added a hidden option for maximum mini player size?